JavaAssignment/src/purva/PurvaAssign3_Calculator_q4_Service.java

```diff
@@ -0,0 +1,51 @@
+/* This is a service class containing all methods which are called by Client class of calculator program */
+package purva;
+
+import java.util.Scanner;
+
+public class PurvaAssign3_Calculator_q4_Service {
+
```

Comment:
Here, you wrote separate - separate scanner class
try write it in main method and method should have parameter so other user/class can also your methods as well.

please remove scanner classes in every method.